File: raspberry/SmartHome/app/src/main/java/smarthome/raspberry/server/httphandlers/ResetPost.java
package smarthome.raspberry.server.httphandlers;

import fi.iki.elonen.NanoHTTPD;
import smarthome.raspberry.model.SmartHomeRepository;

public class ResetPost extends BaseRequestHandler {
    @Override
    public NanoHTTPD.Response serve(NanoHTTPD.IHTTPSession session) {
        SmartHomeRepository.getInstance().removeAll();
        return new NanoHTTPD.Response("Everything is deleted");
    }
}

File: raspberry/SmartHome/app/src/main/java/smarthome/raspberry/server/httphandlers/ResetPost.kt
package smarthome.raspberry.server.httphandlers

import fi.iki.elonen.NanoHTTPD
import smarthome.raspberry.model.SmartHomeRepository

class ResetPost : BaseRequestHandler() {
    override suspend fun serve(session: NanoHTTPD.IHTTPSession): NanoHTTPD.Response {
        SmartHomeRepository.removeAll()
        return NanoHTTPD.Response("Everything is deleted")
    }
}
